pkg/thing/delivery/amqp/client.go

```diff
@@ -144,18 +144,21 @@ func (cs *commandSender) SendListResponse(things []*entities.Thing, corrID strin
 		return err
 	}
 
-	return cs.amqp.PublishPersistentMessage(exchangeDevices, exchangeDevicesType, corrID, msg)
+	return cs.amqp.PublishPersistentMessage(exchangeDevices, exchangeDevicesType, corrID, msg, nil)
 }
 
 // PublishUpdateData send update data command
-func (mp *msgClientPublisher) PublishPublishedData(thingID string, data []entities.Data) error {
-	resp := &network.DataUpdate{ID: thingID, Data: data}
+func (mp *msgClientPublisher) PublishPublishedData(thingID, token string, data []entities.Data) error {
```

Comment:
This comment doesn't match the method's name... could you update it? 😄

Reply:
Done, `vscode` didn't warn me and `golangci-lint` disable it by default. In another PR, I'll enforce this rule and fix some errors. Thanks!